### ISSUE TITLE
在使用infer.py进行推理部署时报错No such file or directory: './export\\termplate_config.json'和verbalizer_config.json

### ISSUE THREAD

**Issue by Zengfanxu1111** [PaddlePaddle/Paddle#55460]
### 请提出你的问题 Please ask your question

   错误出现在infer.py的下面函数中，
 def post_init(self):
        export_path = os.path.dirname(self.args.model_path_prefix)
        template_path = os.path.join(export_path, "template_config.json")
        with open(template_path, "r", encoding="utf-8") as fp:
            prompt = json.load(fp)
            template = AutoTemplate.create_from(prompt, self.tokenizer, self.args.max_length, self.model)
        keywords = template.extract_template_keywords(template.prompt)
        inputs = ["input_ids", "token_type_ids", "position_ids", "attention_mask", "masked_positions"]
        if "soft" in keywords:
            inputs.append("soft_token_ids")
        if "encoder" in keywords:
            inputs.append("encoder_ids")
        verbalizer_path = os.path.join(export_path, "verbalizer_config.json")
        with open(verbalizer_path, "r", encoding="utf-8") as fp:
            label_words = json.load(fp)
            labels = sorted(list(label_words.keys()))

        return template, labels, inputs

我在模型训练完，导出静态图，都没有termplate_config.json'和verbalizer_config.json，请问这两个文件是应该从哪来，烦请大佬解答，新人小白，感激不尽

**Comment by jiahy0825** [comment]
您好，报错信息需要复现代码，请提供代码和环境信息，我们复现后会尽快回复您～
